public/class-monk-public.php

```diff
@@ -218,4 +218,53 @@ public function monk_filter_nav_menus( $args ) {
 
 		return $args;
 	}
+
+	/**
+	 * Function to filter any option
+	 *
+	 * @since    0.5.0
+	 *
+	 * @param  mixed  $pre_option Value to return instead of the option value.
+	 * @param  string $option     String option name.
+	 * @return mixed  $pre_option Value to return instead of the option value.
+	 */
+	public function monk_filter_options( $pre_option, $option ) {
+		$option_names = apply_filters( 'monk_translatable_option', array(
```

Comment:
Rename in plural "monk_translatable_options"

Add documentation to this filter.
https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters